php/Terminus/Models/MachineToken.php

```diff
@@ -17,15 +19,17 @@ public function __construct($attributes, array $options = []) {
 
   /**
    * Deletes machine token
-   *
-   * @return array
+   * @return null
```

Comment:
It doesn't pass the linter without a `@return` tag so I'll add `@return void` for now.